doc/SYNDICATION.md

```diff
@@ -0,0 +1,147 @@
+        
+        ![ft_homepage.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/ft_homepage.png)
+
+        The symbols indicates different syndication grants:
+
+        ![syndication_symbols.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/syndication_symbols.png)
+
+        On clicking on the `green tick` icon, the user can save or download the content
+
+        ![syndication_modal.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/syndication_modal.png)
+
+    - **Republishing Platform**
+
+        On the top right corner on the page, the `Republishing` link takes users to the republishing platform.
+
+        ![republishing_menu.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/republishing_menu.png)
+
+        On the republishing tool, Users can see their contract details, allowance details and select their preferred download format.
+
+        ![republishing_platform.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/republishing_platform.png)
+
+        Users can also manage saved, downloaded and spanish content.
+
+- **Key systems**
+
+    - [**n-syndication**](https://github.com/Financial-Times/n-syndication)
+
+        This is the client side library responsible for adding syndication symbols next to articles/podcast and videos headlines on ft.com. Allowing syndication customers to download or save content for republishing.
+
+        **What it does**
+
+        - Checks if logged in user is a syndication user
+        - Gets all the `contentIds` from the DOM
+
+            ![get_content_ids_snippet.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/get_content_ids_snippet.png)
+
+        - Passes them to [next-syndication-api](https://github.com/Financial-Times/next-syndication-api/tree/master/doc#post-syndicationresolve) to get the syndication data for each content ID
+
+            ![syndication_api_response_snippet.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/syndication_api_response_snippet.png)
+
+        - Uses the response to put the syndication symbols
+        
+    - [**next-syn-list**](https://github.com/Financial-Times/next-syn-list)
+
+        This is the application for the republishing platform.
+
+        The republishing platform allows users to view their contract, manage saved and downloaded content
+
+    - [**next-syndication-api**](https://github.com/Financial-Times/next-syndication-api)
+
+        This is the REST API responsible for powering syndication (Heart of syndication). It lives in heroku with a Postgres DB attached. It handles request from the `n-syndication` and `next-syn-list`. It also runs some cron jobs and sync tasks
```

Comment:
I thought the DB is on AWS? Or is it that the backups live on AWS but the actual DB is on Heroku?

Reply:
It's the latter.

The actual DB is in Heroku. Can be seen on the [dashboard for ft-next-syndication-api on Heroku](https://dashboard.heroku.com/apps/ft-next-syndication-api) under Installed Add-ons section.

The backups are what is stored in an [S3 bucket on AWS](https://github.com/Financial-Times/next-syndication-db-schema#where-are-the-backups).